rainloop/l10n/fr.js

```diff
@@ -0,0 +1,12 @@
+OC.L10N.register(
+    "rainloop",
+    {
+    "Email" : " E-mail",
```

Comment:
If we would like to be very French, we could say "Courriel" but this is quite a debate. That said, I think that Frenchmen rather use "Email" rather than "E-mail"

Reply:
As you wish :) Courriel is the official French word

Reply:
Yes I know and personnaly I would push for "courriel" but I know some people don't like it so I would rather be careful.